lib/syntax.ml

```diff
@@ -178,7 +185,12 @@ let resolve_aggregations =
     | ResInparam _| ResChoices (_, _)
     | ResAggValue _ | ResInTupleList _ -> res_expr
   in
-  handle ~is_agg:false  
+  handle ~is_agg:false 
+  
+let update_schema_with_aliases all_schema final_schema = 
+  List.filter (fun s1 ->
+    List.for_all (fun s2 -> s2.Schema.Source.Attr.attr.name <> s1.Schema.Source.Attr.attr.name) final_schema
```

Comment:
what about unnamed columns btw, `"" <> ""` is it important here?

Reply:
It's function applied to `Column` expression, but unnamed. Like example above but without  alias.
```sql
SELECT COUNT(column_a) FROM test21 WHERE column_a = @column_a
```
If we have more unnamed columns,  the case of uniqueness will always be matched here.